sdks/keyvault/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "azure-keyvault"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Why is this pull request changing the version?

Reply:
Because this is a breaking change. I can revert if you want to, but before release, version should be incremented.

Reply:
I guess I'm curious what our versioning process is going to be.

Reply:
reverting the version, and keep issue pending.